generator/pom.xml

```diff
@@ -150,7 +150,16 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.assertj/assertj-core -->
+        <dependency>
```

Comment:
What does this dependency introduce into the project?
What is the licence for the dependency?

Reply:
assertj allows fluent syntax to be used to write assertions when unit testing. @r-stuart has used it previously, and has used it here to write some of his unit tests.
See: https://github.com/joel-costigliola/assertj-core

The licence is Apache 2.0 see: https://github.com/joel-costigliola/assertj-core/blob/master/LICENSE.md

Reply:
Dependency removed